test/system/submit_pledge_test.rb

```diff
@@ -69,6 +72,35 @@ class SubmitPledgeTest < ApplicationSystemTestCase
     end
   end
 
+  describe 'clinic contact info' do
+    before do
+      @clinic.email_for_pledges = "pledges@catfund.biz"
+
+      @patient.update pledge_sent: false
+      
+      visit edit_patient_path @patient
+      wait_for_element 'Patient information'
+    end
+
+    it 'should show clinic email instead of fax' do
+      find('#submit-pledge-button').click
+      wait_for_element 'Patient name'
+      assert has_text? 'Confirm the following information is correct'
+      find('#pledge-next').click
+      wait_for_ajax
+
+      wait_for_no_element 'Confirm the following information is correct'
+      assert has_text? 'Generate your pledge form'
+      find('#pledge-next').click
+      wait_for_no_element 'Review this preview of your pledge'
+
+      assert has_text? 'Awesome, you generated a CATF'
+      # now we should see the email
+      assert has_text? "pledges@catfund.biz"
+      refute has_text? "Fax service"
+    end
+  end
```

Comment:
why doesn't this test work!

Reply:
to follow up – if i put a `byebug` breakpoint inside the test body, i can confirm that, indeed, `@patient.clinic.email_for_pledges` has the correct value. _but then for some reason_ the view isn't picking up that value! See [here](https://github.com/DARIAEngineering/dcaf_case_management/compare/eli/clinic-email#diff-e3fbc4b423f7b6afa5f19a9d23e9208f91d1740b7723fbd3f3a32635be9c11edR41-R54) for what the view is doing.

Reply:
massive h/t @FeminismIsAwesome for reminding me that models have to be saved!!! drrrrrrr